edxval/tests/test_api.py

```diff
@@ -1306,8 +1306,10 @@ def test_get_videos_for_ids(self):
         video_data = list(video_data_generator)[0]
         self.assertEqual(video_data['courses'][0]['test-course'], self.image_url)
 
+    # attr_class is django magic: https://github.com/django/django/blob/master/django/db/models/fields/files.py#L214
+    @patch('edxval.models.CustomizableImageField.attr_class.save', side_effect = Exception("pretend save doesn't work"))
```

Comment:
Hah, welcome to my life 6 hours ago!  The `FieldFile.save()` method wants the `size` attribute of whatever you're saving, and a file object doesn't have that attribute:
https://github.com/django/django/blob/stable/1.8.x/django/db/models/fields/files.py#L106